docs/workforce-management-framework/topics/con-demo-app.adoc

```diff
@@ -0,0 +1,6 @@
+[id='con-demo-app-{chapter}']
```

Comment:
I'm not convinced of idea of spliting this content. There is zero chance to reuse it and we may risk that it will be detached from context. I like idea of modularized documentation but let's avoid extracting 2 sentences into separate file. 

ping @austincunningham 

ping @austincunningham for opi